addons/hivedr/src/main/java/org/apache/falcon/hive/HiveDROptions.java

```diff
@@ -147,6 +147,11 @@ public String getExecutionStage() {
         return context.get(HiveDRArgs.EXECUTION_STAGE);
     }
 
+    public boolean isTDEEncryptionEnabled() {
+        return StringUtils.isEmpty(context.get(HiveDRArgs.TDE_ENCRYPTION_ENABLED))
+                ? false : Boolean.valueOf(context.get(HiveDRArgs.TDE_ENCRYPTION_ENABLED));
```

Comment:
if the string has a typo like "flase" , the parse would throw an error. If the string is not defined, we want to use the default value, which is false.